tgui/LICENSE.md

```diff
@@ -1,3 +1,5 @@
+MIT license

```

Comment:
Maybe not, but it's the MIT license
https://opensource.org/licenses/mit-license.html